pip/commands/list.py

```diff
@@ -146,15 +153,20 @@ def run(self, options, args):
             raise CommandError(
                 "Options --outdated and --uptodate cannot be combined.")
 
+        # TODO (nvdv): Check options combinations.
+
```

Comment:
I'd say we either implement the option completely (in combination with other options) or `pip list` should emit a warning saying explicitely (or crash like for --outdated/--uptodate) when the option is ignored.
(it does not seem hard to implement it completely though).